bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -178,11 +173,11 @@ def find_spec(self, fullname: str, path, target=None) -> ModuleSpec:
 
     ip = get_ipython()
     if hasattr(ip, "kernel"):
-        import JupyROOT
+        import JupyROOT  # noqa: F401  # imported the side effect of setting up JupyROOT
```

Comment:
I don't understand the comment here. If it is about the lazy import at function scope, why don't we need the same noqa in other places?

And more in general, is the linter happy with lazy imports at function scope?

Reply:
Yes, the linter is generally happy with lazy imports at function scope.

The warning suppression here is not about imports. It's about the unused module `JupyROOT`, which is fine because we import the module not to use is, but for the side effects in `JupyROOT/__init__.py`.

Reply:
Thank you, I really missed the point of the comment, thanks for the explanation!